schemas/manifest.schema.json

```diff
@@ -0,0 +1,455 @@
+                "fund": {"type": "string", "format": "uri-reference"},
+                "cpe": {
+                    "type": "string",
+                    "pattern": "^cpe:[0-9\\.]*:[a-z]*:[a-zA-Z0-9_]*:[a-z]*$"
+                }
+            }
+        },
+        "integration": {
+            "type": "object",
+            "required": ["yunohost", "architectures", "multi_instance", "ldap", "sso"],
+            "additionalProperties": false,
+            "properties": {
+                "yunohost": {
+                    "type": "string",
+                    "pattern": "^>= ?[0-9]*\\.[0-9]*\\.[0-9]*"
+                },
+                "architectures": {
+                    "oneOf": [
+                        {
+                            "type": "string",
+                            "const": "all"
+                        },
+                        {
+                            "type": "array",
+                            "items": {
+                                "type": "string",
+                                "enum": ["amd64", "i386", "armhf", "arm64"]
+                            }
+                        }
+                    ]
+                },
+                "multi_instance": {"$ref": "#/$defs/boolean_or_not_relevant"},
+                "ldap": {"$ref": "#/$defs/boolean_or_not_relevant"},
+                "sso": {"$ref": "#/$defs/boolean_or_not_relevant"},
+                "disk": {"$ref": "#/$defs/byte_size"},
+                "ram": {
+                    "type": "object",
+                    "required": [],
+                    "additionalProperties": false,
+                    "properties": {
+                        "build": {"$ref": "#/$defs/byte_size"},
+                        "runtime": {"$ref": "#/$defs/byte_size"}
+                    }
+                }
+            }
+        },
+        "install": {
+            "type": "object",
+            "required": [],
+            "additionalProperties": false,
+            "patternProperties": {
+                "^[a-z][a-z0-9_]*$": {
+                    "type": "object",
+                    "required": ["type"],
+                    "additionalProperties": false,
+                    "properties": {
+                        "type": {
+                            "type": "string",
+                            "enum": [
+                                "string", "text", "select", "tags", "email", "url", "date", "time",
+                                "color", "password", "path", "boolean", "domain", "user", "group",
+                                "number", "range", "alert", "markdown", "file", "app"
+                            ]
+                        },
+                        "default": {
+                            "type": ["string", "number", "boolean"]
+                        },
+                        "ask": {"$ref": "#/$defs/translated_string"},
+                        "help": {"$ref": "#/$defs/translated_string"},
```

Comment:
So eeeh we should also have `example` in here I think, which is often use (though less than before because it's not relevant for many stuff)

Reply:
Aaaand possibly an `optional` = True or False key ?

Reply:
Fixed : 
```
                        "example": {"type": ["string", "number", "boolean"]},
                        "optional": {"type": "boolean"},
```